config/coordinator/log4j2-dev.xml

```diff
@@ -112,7 +112,7 @@
       <DebouncingFilter/>
       <appender-ref ref="rewrite"/>
     </Logger>
-    <Logger name="clients.l1" level="INFO" additivity="false">
+    <Logger name="clients.l2" level="DEBUG" additivity="false">
```

Comment:
Why stop printing`clients.l1` info log anymore?

Reply:
root log level is in INFO, it will print it. I added DEBUG for L2 to log failures